docs/sources/configure-client/_index.md

```diff
@@ -11,9 +11,12 @@ weight: 300
 # Configure the client to send profiles
 
 Pyroscope is a continuous profiling database that allows you to analyze the performance of your applications.
-When sending profiles to Pyroscope, you can choose between two primary methods: SDK instrumentation and auto-instrumentation using Grafana Alloy.
+When sending profiles to Pyroscope, you can choose between three methods:
+1. Auto-instrumentation using Grafana Alloy
+2. SDK instrumentation
+3. SDK instrumentation through Grafana Alloy
 
-This document explains these two techniques and guide you when to choose each one.
+This document explains these techniques and guide you when to choose each one.
 
 ![Pyroscope agent server diagram](https://grafana.com/media/docs/pyroscope/pyroscope_client_server_diagram_09_18_2024.png)
```

Comment:
This needs to be updated with the new image @Rperry2174

Reply:
just put in a request with design team sorry for delay!